flo_web/web_app/src/components/robotControl/GameContainer.tsx

```diff
@@ -267,13 +271,19 @@ const GameContainer: React.FunctionComponent<GameContainerProps> = ({
               console.error("tried to play game when not connected");
               return;
             }
-            if (buckets[gbID] === undefined) {
-              console.error("tried to play a game with a bad game id: " + gbID);
-              return;
+            let steps = [] as StepDef[];
+            if (["simon_says", "target_type"].includes(gameType)) {
```

Comment:
This line is a possible source of the regression. Should be target_touch not target_type